python/py-mmtf-python/Portfile

```diff
@@ -33,4 +33,9 @@ if {${name} ne ${subport}} {
     depends_build-append    port:py${python.version}-setuptools
     depends_lib-append      port:py${python.version}-msgpack
     livecheck.type          none
+
+    depends_test-append     port:py${python.version}-nose
```

Comment:
`numpy` is also a test dependency - it fails without

Reply:
I cannot comment on that part as you haven't changed anything there now. But, `master_sites` and `distname` are the defaults that are already set by the `python` PG so you can remove them.

Reply:
How did you find the numpy dependence? Thanks anyway!

Reply:
well... it's listed in their requirements file (but indeed not in the setup.py as a test dependency). It's typically a good idea to perform the installation (and the tests) in trace mode, i.e., `sudo port -dvt test <>`, so that it will be only allowed to use packages that you have explicitly listed as dependencies. In this case you'll also see `import numpy` in the tests directory, which implies it's needed here.

Reply:
thanks!